pages/home/index.js

```diff
@@ -21,13 +22,30 @@ function Home() {
       <Nav email={AuthUser.email} signOut={AuthUser.signOut} />
 
       <main>
-        <MainDisplay/>
+        <MainDisplay claims={AuthUser.claims} />
       </main>
     </div>
   );
 }
 
-export const getServerSideProps = withAuthUserTokenSSR()();
+export const getServerSideProps = withAuthUserTokenSSR()(
+  async ({ AuthUser }) => {
+    const user = await getFirebaseAdmin().auth().getUserByEmail(AuthUser.email);
+    if (user.email == "lmsplatformcscix691@gmail.com") {
```

Comment:
setting all users other than the lms email user to a student -> we only have one admin so far this can be changed and its also hardcoded

Reply:
this is probably what should be happening in the admin panel yes? Like this code here can be used to set user permissions (the point of our admin panel), we can just get redo the hardcoded logic and some other functionality to it

Reply:
somewhat yes, but the admin panel isnt the first thing that every user sees meaning the user will never get set to have a student role unless and admin logs in

I think it would need to be both here and in an admin panel